src/System.Data.Common/tests/System/Data/XmlDataReaderTest.cs

```
@@ -335,6 +321,7 @@ void IXmlSerializable.ReadXml(XmlReader reader)
             {
                 XmlDocument doc = new XmlDocument();
                 string str = reader.ReadString();
+                //TODO: what
                 try
```

swallowing all exceptions?